.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: erlang
 sudo: false
 otp_release:
-  - 20.0
-  - 19.3
+  - 23.0
+  - 22.3
+  - 21.3
```

Comment:
will 21.0 still be supported right?

Reply:
Yes, I have set the min otp version to 21.

Travis gave build errors with 19 and 20, so I assume that they don't support the new `ei` interfaces.

Reply:
Yeah, I'm not quite sure when all were introduced. I thought it was on 20 but I may be wrong

Reply:
20 sounds like what we see in the Travis results.

I will call this release 1.1.0 and give a shout out at the erlang-questions mailing list.